MAINTAINERS

```diff
@@ -1,69 +1,65 @@
-# Official list of OpenEBS Maintainers.
+# Official list of OpenEBS Maintainers and Reviewers.
 #
-# Names added to this file should be in the following format:
-#     Individual's name,@githubhandle, Company Name
+# Reflecting structural changes to the project with respect
```

Comment:
I would also request to have a change saying that - only the addition need to be happening to emeritus list but not to be removed.

Reply:
Sorry @vishnuitta but I don't think I understand what specific change it is that you are requesting.

Are you proposing that we include a statement to the effect that once maintainers have been transitioned to emeritus status, they cannot be removed from it (other than by restoration to active maintainer status)?  If so, then I think if enacted that would form a part of project governance policy and therefore should be added to the appropriate document(s), not the list itself.  That would make its treatment consistent with that for the election and withdrawal of other recognised roles.

Or do I misunderstand?

Reply:
you got it right @GlennBullingham . Can we have that project governance policy update some apt place in the repo?
All I'm trying to convey is: Lets make sure that due credits are given to the previous contributors even in the future.

Reply:
@vishnuitta I'm in complete and vehement agreement with you on that.  Due credit should given in perpetuity and to me at least, that's the purpose of the emeritus list(s), to call out those persons who have made valued contributions but have, as priorities change, ceased to actively engage.  In due course I will draft what I think to be an appropriate amendment / addendum on that matter to the current policy and solicit your review, that of the active maintainers (such as that list should end up, at the merging of this PR), and from all persons who contribute a review to this PR.